test/transcoding/block_w_struct_return.cl

```diff
@@ -1,17 +1,7 @@
 // RUN: %clang_cc1 -triple spir -cl-std=cl2.0 -disable-llvm-passes -fdeclare-opencl-builtins -finclude-default-header %s -emit-llvm-bc -o %t.bc
-// TODO: currently max version is limited to 1.1 for this test. Issues here
```

Comment:
Can we modify this test in a way that it will check both cases when max version is limited to 1.1 and when it is not limited at all?